src/components/team/Teammate.jsx

```diff
@@ -1,5 +1,34 @@
+import React from "react";
+import Image from "next/image";
+
 const Teammate = () => {
-  return <div>Teammate</div>;
+  return (
+    <div className="bg-gradient-to-r from-hlg-blue-300 to-hlg-black ...">
```

Comment:
i dont think "..." does anything unless you have the "truncate" style which doesn't make sense to use here?